models/LinearSwitcher/ls.jl

```diff
@@ -0,0 +1,106 @@
+using Reachability, HybridSystems, MathematicalSystems, LazySets, LinearAlgebra, SX, SymEngine
+using Plots, LaTeXStrings
+
+file = "models/LinearSwitcher/model.xml"
+model = readsxmodel(file, raw_dict=true)
+
+function linear_switching(; X0 = Singleton([3.1, 4.0, 0.0, 0.0, 0.0]),
+                            U = Interval(-1.0, 1.0),
+                            T = 1.0,
+                            ε = 1e-6)
+
+    n = 5 # number of state variables
+    m = 1 # number of input variables
+    state_vars = convert.(Basic, [fi.args[1].args[1] for fi in model["flows"][1]])
+    input_vars = [convert(Basic, collect(keys(model["variables"]))[6])] # same as convert(Basic, :u)
+
+    id_location = 1
+    A, B, c = get_coeffs(model["flows"][id_location], n, m, state_vars, input_vars) # new function in SX
+    X = HPolyhedron([HalfSpace([-1.0, 0.0, 0.0, 0.0, 0.0], -3.0 + ε)]) # x >= 3
```

Comment:
```suggestion
    X = HalfSpace([-1.0, 0.0, 0.0, 0.0, 0.0], -3.0 + ε)  # x1 >= 3
```
same for the other invariants

Reply:
One part of the suggestion was to rename `x` → `x1` :wink: